columnphysics/icepack_therm_vertical.F90

```diff
@@ -241,7 +242,7 @@ subroutine thermo_vertical (nilyr,       nslyr,     &
          einter          ! intermediate energy
 
       real (kind=dbl_kind) :: &
-         fadvocn ! advective heat flux to ocean
+         fadvocn, saltvol, dfsalt ! advective heat flux to ocean
```

Comment:
Put the new variables on separate lines so that they can be identified with ! comments (add units too please)